misc/python/materialize/mzbuild.py

```diff
@@ -208,21 +212,26 @@ def __init__(self, rd: RepositoryDetails, path: Path, config: Dict[str, Any]):
         self.strip = config.pop("strip", True)
         self.extract = config.pop("extract", {})
         self.rustflags = config.pop("rustflags", [])
+        self.channel = config.pop("channel", None)
```

Comment:
Can we do without this for now? It seems weird to me to allow an `mzbuild.yml` to specify what channel to use; it seems like something that should always be in the control of the person running `mzimage` or `mzcompose`.

Reply:
Like, I think should just be `self.channel = "stable"` or `self.channel = None` unless there's some reason to configure this on a per-image basis that I'm not seeing!